package.json

```diff
@@ -86,7 +86,7 @@
     "@playwright/test": "^1.31.2",
     "@sanity/client": "^6.1.1",
     "@sanity/pkg-utils": "^2.1.0",
-    "@sanity/tsdoc": "1.0.0-alpha.22",
+    "@sanity/tsdoc": "1.0.0-alpha.25",
     "@sanity/tsdoc-to-portable-text": "^0.4.3",
```

Comment:
❓Looks like `@sanity/tsdoc-to-portable-text` is unused, is that needed?

Reply:
I would be for removing it but I'm also unsure if it's being used somewhere hidden. @mariuslundgard any reason to keep it?

Reply:
Should be safe to remove – it's an older incarnation of `@sanity/tsdoc`